src/types/User.ts

```diff
@@ -32,6 +34,11 @@ export interface ProfileLink {
   url: string;
 }
 
+export interface OnboardingData {
+  perTemplate?: Partial<Record<VenueTemplate, number>>;
+  perVenue?: Record<string, number>;
+}
+
```

Comment:
Data saved in the user profile in DB that holds the timestamps of when a user finished onboarding per template and per venue